plugins/chat/app/services/chat/create_message.rb

```diff
@@ -34,6 +34,8 @@ class CreateMessage
     end
 
     policy :no_silenced_user
+    policy :accept_blocks
+
```

Comment:
No need for the extra blank line?

Reply:
not sure what extra blank line you are talking about, if you talk of the line before `params {}`, I like it 🤷‍♂️

Reply:
Yes, that’s true that with multi-line blocks, it can feel a bit weird.